static/app/components/organizations/hybridFilter.tsx

```diff
@@ -43,7 +44,7 @@ export interface HybridFilterProps<Value extends SelectKey>
   onChange: (selected: Value[]) => void;
   value: Value[];
   checkboxWrapper?: (
-    renderCheckbox: (props: CheckboxProps) => React.ReactNode
+    renderCheckbox: (props: {disabled?: boolean}) => React.ReactNode
```

Comment:
narrowed the interface because there is only one usage and it only passes `disabled`